### PR TITLE
Handle muliple processes

### PR DESCRIPTION
**Pull request by diegiesskanne**
Backend implementation for multiple processes and Redis Setup

### CODE REVIEW

**Comment by diegiesskanne** [comment]
The merge of this branch removes the ui from the service dashboard. But the backend is more robust and can handle multiple emba processes at the same time. Ravi is currently working on the new frontend part.

File: docker-compose.yml [hunk 1]
```diff
@@ -6,9 +6,9 @@ services:
     restart: always
     env_file:
       - .env
-    network_mode: "host"
     ports:
       - '3306:3306'
+
```
Comment (ashutoshsingh0223) [inline docker-compose.yml:11]: Add `network_mode` directive

Comment (diegiesskanne) [reply docker-compose.yml:11]: host-mode?

Comment (diegiesskanne) [reply docker-compose.yml:12]: resolved

**Review by MaximilianWagner — APPROVED** [review]
Looks good.
Maybe mark as work in progress.
Don't forget the testcases ^^

**Comment by diegiesskanne** [comment]
do not merge
this still needs work

**Comment by diegiesskanne** [comment]
can be merged now in my opinion

**Pull request merged**